aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -1644,6 +1614,83 @@ static at::Tensor _quantized_convolution_onednn(
     unary_scalars,
     unary_algorithm.has_value() ? unary_algorithm.value() : ""
   );
+
+#if IDEEP_PREREQ(3, 1, 0, 0)
+  // Use oneDNN's APIs instead of prepare/compute from ideep to reduce framework overhead.
```

Comment:
```suggestion
  // Use oneDNN's APIs instead of prepare/compute from ideep to reduce integration overhead.
```

Reply:
Thanks. I have modified the comments here. The overhead reduction is not due to **new** oneDNN API. It's because ideep is too heavy. Using old oneDNN should be similar.

Reply:
which part is heavy? Can you elaborate?

Reply:
It's mainly about the `prepare` part. Since we need to keep the ideep API unchanged when upgrading oneDNN from v2.x to v3.x with many API changes, such as `dnnl::primitive_attr`, ideep has to make its own data structures more complex to preserve the same behavior as before. And ideep also has to deal with edge cases with a lot of conditional branches.

Reply:
I have modified the comments in the code as below:
```cpp
  // Use oneDNN's APIs instead of prepare/compute from ideep to reduce integration overhead.
  // The functions from ideep are heavy because they have complex data structures for unified API
```